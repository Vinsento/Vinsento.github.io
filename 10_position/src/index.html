<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="initial-scale=1.0, width=device-width">
	<title>Who we are</title>
	<link rel="shortcut icon" type="image/x-icon" href="assets/fav/favicon.ico">
	<link rel="stylesheet" href="css/style.css">
</head>
<body>

	<div class="container">
		<h1 class="title">who we are</h1>

		<p class="subtitle">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Labore aperiam fugit consequuntur, voluptatibus ex sint iure in distinctio sed dolorem.</p>

		<ul class="list">
			<li class="list__item">
				<div class="list__link">
					<div class="list__photo">
						<img src="img/01.jpg" alt="Jon`s photo" class="list__img">
						<a href="#" class="social left_40">
							<img src="img/tw.png" alt="twitter">
						</a>
						<a href="#" class="social">
							<img src="img/fb.png" alt="facebook">
						</a>
						<a href="#" class="social left_60">
							<img src="img/g.png" alt="google">
						</a>
					</div>
					<div class="list__about">
						<p class="name">john doe</p>
						<p class="post">team lead</p>
					</div>
				</div>
			</li>
			<li class="list__item">
				<div class="list__link">
					<div class="list__photo">
						<img src="img/03.jpg" alt="Alex photo" class="list__img">
						<a href="#" class="social left_40">
							<img src="img/tw.png" alt="twitter">
						</a>
						<a href="#" class="social">
							<img src="img/fb.png" alt="facebook">
						</a>
						<a href="#" class="social left_60">
							<img src="img/g.png" alt="google">
						</a>
					</div>
					<div class="list__about">
						<p class="name">alex cutter</p>
						<p class="post">developer</p>
					</div>
				</div>
			</li>
			<li class="list__item">
				<div class="list__link">
					<div class="list__photo">
						<img src="img/04.jpg" alt="Elena photo" class="list__img">
						<a href="#" class="social left_40">
							<img src="img/tw.png" alt="twitter">
						</a>
						<a href="#" class="social">
							<img src="img/fb.png" alt="facebook">
						</a>
						<a href="#" class="social left_60">
							<img src="img/g.png" alt="google">
						</a>
					</div>
					<div class="list__about">
						<p class="name">elena simpson</p>
						<p class="post">artist</p>
					</div>
				</div>
			</li>
			<li class="list__item">
				<div class="list__link">
					<div class="list__photo">
						<img src="img/05.jpg" alt="Lily photo" class="list__img">
						<a href="#" class="social left_40">
							<img src="img/tw.png" alt="twitter">
						</a>
						<a href="#" class="social">
							<img src="img/fb.png" alt="facebook">
						</a>
						<a href="#" class="social left_60">
							<img src="img/g.png" alt="google">
						</a>
					</div>
					<div class="list__about">
						<p class="name">lily allen</p>
						<p class="post">desinger</p>
					</div>
				</div>
			</li>
		</ul>
	</div>

	<script src="js/bundle.js"></script>
</body>
</html>
