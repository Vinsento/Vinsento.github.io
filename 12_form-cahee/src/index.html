<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="initial-scale=1.0, width=device-width">
	<title>Form cahee</title>
	<link rel="shortcut icon" type="image/x-icon" href="assets/fav/favicon.png">
	<link rel="stylesheet" href="css/style.css">
</head>
<body>

	<div class="content">

		<div class="wrap">
			<header class="header">
				<h1 class="header__title">contact</h1>
				<p class="header__subtitle">For new projects, ideas, business or anything you'd like to discuss, contact with me!</p>
			</header>
			
			<section class="main">
				<form action="#" class="form">
					<input type="text" placeholder="Enter your name" class="form__input">
					<input type="email" placeholder="Enter your mail" class="form__input">
					<input type="tel" placeholder="Enter your phone" class="form__input">
					<textarea name="message" cols="30" rows="10" placeholder="Enter your message" class="form__input textarea"></textarea>
				</form>
				<div class="info">
					<h5 class="info__title">addres</h5>
					<address class="info__subtitle">123 City name, Street, City, Country</address>
					<h5 class="info__title">email</h5>
					<a href="mailto:cahee@domain.com" class="info__subtitle">cahee@domain.com</a>
					<h5 class="info__title">phone</h5>
					<a href="tel:+0123456789" class="info__subtitle">+012 345 6789</a>
				</div>
			</section>
		</div>

	</div>
	<footer class="footer">
		<p class="footer__text">Â© 2015 Company. All right reserved.</p>
		<a href="#" class="footer__anchor">cahee</a>
		<p class="footer__text">Made with <span class="red">love</span> by Auskteez</p>
	</footer>
	<script src="js/bundle.js"></script>
</body>
</html>
