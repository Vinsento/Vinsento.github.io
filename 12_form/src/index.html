<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="initial-scale=1.0, width=device-width">
	<title>Title</title>
	<link rel="shortcut icon" type="image/x-icon" href="assets/fav/favicon.ico">
	<link rel="stylesheet" href="css/style.css">
</head>
<body>

	<div class="wrap">
		<form action="#" class="form">
			<div class="form__roof">
				<input type="radio" name="option" class="form__check" id="one" checked>
				<label for="one" class="form__actions">Sign in</label>
				<input type="radio" name="option" class="form__check" id="two">
				<label for="two" class="form__actions">Join us</label>
			</div>
			<label>
				<span class="form__field">username</span>
				<input type="text" class="form__input">
			</label>
			<label>
				<span class="form__field">password</span>
				<input type="password" class="form__input">
			</label>
			<div>
				<input type="checkbox" id="check">
				<label for="check" class="form__checkbox">Keep me signed in</label>
			</div>
			<input type="submit" value="Sign in" class="form__submit">

			<div class="form__basement">
				<a href="#" class="form__link">Forgot password</a>
				<a href="#" class="form__link">Create an account</a>
			</div>
		</form>
	</div>

	<script src="js/bundle.js"></script>
</body>
</html>
