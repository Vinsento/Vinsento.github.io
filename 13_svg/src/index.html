<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="initial-scale=1.0, width=device-width">
	<title>Design Lab</title>
	<link rel="shortcut icon" type="image/x-icon" href="assets/fav/favicon.png">
	<link rel="stylesheet" href="css/style.css">
</head>
<body>

	<div class="first-screen">
		<img src="./img/logo.svg" alt="logo" class="first-screen__logo">
		<h1 class="first-screen__title"><span class="db">working</span> with design lab</h1>
		<p class="first-screen__subtitle">
			<span class="db">We’ve been helping startups launch and grow their businesses.</span>
			<span class="db">We help you to fill the gap between development,</span>
			design and product management.
		</p>
		<a href="#" class="first-screen__scroll">
			<img src="./img/scroll.svg" alt="scroll">
		</a>
	</div>

	<section class="about">
		<div class="about__wrap">
			<h2 class="about__title">about us</h2>
			<p class="about__text">When you work with us, you are part of a team of committed collaborators. <span class="db">All aspects of our software development process are integrated, iterative and agile.</span></p>
			<p class="about__description">Our approach unifies design, development and product management to create exceptional products. We start each project by understanding your business goals, the user’s needs and the product requirements. Then we help translate them into effective and elegant solutions.</p>
		</div>
	</section>

	<section class="services">
		<div class="services__wrap">
			<h2 class="services__title">services</h2>
			<p class="services__text">Before we sign a contract or write a line of code our team will spend the necessary time needed to understand your product vision.</p>
			<p class="services__text">The outcome of this scoping session is a proposal or high-level statement of work.</p>
			<p class="services__text">Based on your requirements, we can provide a complete offering or just the pieces you need.</p>
		</div>
	</section>

	<section class="what-we-do">
		<div class="what-we-do__wrap">
			<h2 class="what-we-do__title">what we do</h2>
			<div class="what-we-do__item">
				<figure class="what-we-do__figure">
					<img src="./img/pen.png" alt="pen" class="what-we-do__img">
					<figcaption class="what-we-do__caption">desing</figcaption>
				</figure>
				<p class="what-we-do__text">Our design practice offers a full range of services including branding strategy,
				interaction and visual design and user experience testing. Throughout your project, our designers create and implement visual designs and workflows, solicit user feedback, and work with you to make sure what gets built is what is needed. </p>
			</div>
			<div class="what-we-do__item">
				<figure class="what-we-do__figure">
					<img src="./img/tag.png" alt="tag" class="what-we-do__img">
					<figcaption class="what-we-do__caption">development</figcaption>
				</figure>
				<p class="what-we-do__text">All engineers are fluent in the latest enterprise, mobile and web development technologies. They collaborate with your team to write, test, and improve code on a daily basis, using proven practices such as test-driven development and pair programing.</p>
			</div>
			<div class="what-we-do__item">
				<figure class="what-we-do__figure">
					<img src="./img/table.png" alt="table" class="what-we-do__img">
					<figcaption class="what-we-do__caption">product management</figcaption>
				</figure>
				<p class="what-we-do__text">Planning and development is iterative. Because we are constantly coding and testing, the products we build are always ready to go live. This iterative process allows for changes as business requirements evolve.</p>
			</div>
		</div>
	</section>

	<section class="our-works">
		<ul class="our-works__list">
			<li class="our-works__item">
				<a href="#" class="our-works__link first"><span>topworkz</span></a>
			</li>
			<li class="our-works__item">
				<a href="#" class="our-works__link two"><span>joint</span></a>
			</li>
			<li class="our-works__item">
				<a href="#" class="our-works__link three"><span>market</span></a>
			</li>
			<li class="our-works__item">
				<a href="#" class="our-works__link four"><span>worktop</span></a>
			</li>
			<li class="our-works__item">
				<a href="#" class="our-works__link five"><span>label</span></a>
			</li>
			<li class="our-works__item">
				<a href="#" class="our-works__link six"><span>kyiv</span></a>
			</li>
		</ul>
	</section>

	<section class="contact">
		<div class="contact__wrap">
			<h2 class="contact__title">contact us</h2>
			<p class="contact__text">Planning and development is iterative. Because we are constantly coding and testing, the products we build are always ready to go live. This iterative process allows for changes as business requirements evolve.</p>
			<a href="#" class="contact__btn">contact us</a>
		</div>
	</section>

	<footer class="footer">
		<svg id="logo.svg" xmlns="http://www.w3.org/2000/svg" width="35" height="37" viewBox="0 0 69 71" class="footer__img">
			<defs>
				<style>
					.cls-1, .cls-2 {
						fill: none;
						stroke-width: 2px;
					}

					.cls-1, .cls-2 {
						stroke: #000;
					}

					.cls-1, .cls-2, .cls-3 {
						fill-rule: evenodd;
					}

					.cls-3, .cls-4 {
						fill: #000;
					}
				</style>
			</defs>
			<path class="cls-1" d="M990.675,352.857l-28.815-55-28.814,55h57.629Z" transform="translate(-927 -284)"/>
			<path class="cls-2" d="M961.5,285A33.216,33.216,0,1,1,928,318.214,33.357,33.357,0,0,1,961.5,285Z" transform="translate(-927 -284)"/>
			<path id="Ellipse_1_copy_8" data-name="Ellipse 1 copy 8" class="cls-3" d="M961.86,347.857a3.572,3.572,0,1,1-3.6,3.572A3.587,3.587,0,0,1,961.86,347.857Z" transform="translate(-927 -284)"/>
			<circle id="Ellipse_1_copy_9" data-name="Ellipse 1 copy 9" class="cls-4" cx="61.515" cy="67.422" r="3.61"/>
			<path id="Ellipse_1_copy_11" data-name="Ellipse 1 copy 11" class="cls-3" d="M933.043,347.857a3.572,3.572,0,1,1-3.6,3.572A3.587,3.587,0,0,1,933.043,347.857Z" transform="translate(-927 -284)"/>
		</svg>

		<p>© 2014. All Rights Reserved Ellipsis.com</p>
	</footer>



	<script src="js/bundle.js"></script>
</body>
</html>
