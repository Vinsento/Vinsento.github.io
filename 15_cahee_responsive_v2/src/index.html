<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="initial-scale=1.0, width=device-width">
	<title>cahee</title>
	<link rel="shortcut icon" type="image/x-icon" href="assets/fav/favicon.ico">
	<link rel="stylesheet" href="css/style.css">
</head>
<body>

	
	<div class="content">
		
		<div class="first-screen">
			<div class="header">
				<a href="#" class="header__logo">cahee</a>
				<nav class="header__nav">
					<ul class="nav__list">
						<li class="nav__item">
							<a href="#" class="nav__btn">home</a>
						</li>
						<li class="nav__item">
							<a href="#" class="nav__btn">about</a>
						</li>
						<li class="nav__item">
							<a href="#" class="nav__btn">services</a>
						</li>
						<li class="nav__item">
							<a href="#" class="nav__btn">galerry</a>
						</li>
						<li class="nav__item">
							<a href="#" class="nav__btn">blog</a>
						</li>
						<li class="nav__item">
							<a href="#" class="nav__btn">contact</a>
						</li>
					</ul>
				</nav>
				<a href="#" class="nav__menu-btn">
					<span class="burger-line"></span>
				</a>
			</div>

			<section class="home">
				<h2 class="home__title">fresh life</h2>
				<p class="home__text">Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco.</p>
				<a href="#" class="home__btn">get free now</a>
			</section>
		</div>
	</div>

	<section class="about">
		<img src="./img/berry.png" alt="berry">
		<div class="about__info">
			<h1 class="about__title">about us</h1>
			<p class="about__text">Abore Cahee magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum.</p>
			<p>Nim id est laborum. Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore.</p>
		</div>
	</section>





	<script src="js/bundle.js"></script>
</body>
</html>
