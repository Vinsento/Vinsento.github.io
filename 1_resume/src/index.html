<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="initial-scale=1.0, width=device-width">
	<title>Title</title>
	<link rel="shortcut icon" type="image/x-icon" href="assets/fav/favicon.ico">
	<link rel="stylesheet" href="css/style.css">
</head>
<body>
	<header>
		<div class="header_content">
			<div class="header_content_button">
				<a href="_index.html" class="header_content_button_anchor">язык</a>
			</div>
			<div class="header_content_name">
				<p>Ксенз Виталий</p>
			</div>
			<div class="header_content_photo">
				<img src="./img/IMG_0001.jpg" alt="user photo">
			</div>
		</div>
	</header>
	<main>
		<div class="main_content">
			<div class="main_content_left-column">
				<section class="text_content">
					<div class="text_content_header">
						<p>Персональная информация</p>
					</div>
					<div class="text_content_main">
						<p>Имя: Ксенз Виталий</p>
						<p>Год рождения: 27.11.1987г.</p>
						<p>Город: Кременчуг</p>
						<p>Семейное положение: не женат</p>
					</div>
				</section>
				<section class="text_content">
					<div class="text_content_header">
						<p>Контактные данные</p>
					</div>
					<div class="text_content_main">
						<p>Телефон: +38 093 781 50 08</p>
						<p>Почта: hummanist@gmail.com</p>
					</div>
				</section>
				<section class="text_content">
					<div class="text_content_header">
						<p>Опыт работы</p>
					</div>
					<div class="text_content_main">
						<p>2019г. - настоящее время: биржа фриланс.</p>
						<p>2012г. - 2019г. торговый представитель.</p>
						<p>2010г. - 2012г. механик участка ПГОК.</p>
					</div>
				</section>
			</div>
			<div class="main_content_right-column">
				<section class="text_content">
					<div class="text_content_header">
						<p>Образование</p>
					</div>
					<div class="text_content_main">
						<p>2005г. - 2010г. КНУ им. Остроградского.</p>
						<p>Факультет: Технология машиностроения.</p>
					</div>
				</section>
				<section class="text_content">
					<div class="text_content_header">
						<p>Дополнительное обучение</p>
					</div>
					<div class="text_content_main">
						<p>сентябрь 2019г. - декабрь 2019г. курсы Beetroot Academy</p>
						<p>Направление: Front-end разаработка.</p>
					</div>
				</section>
				<section class="text_content">
					<div class="text_content_header">
						<p>Навыки</p>
					</div>
					<div class="text_content_main">
						<p>CSS</p>
						<p>HTML</p>
						<p>JavaScript</p>
						<p>Git</p>
					</div>
				</section>
			</div>
		</div>
	</main>

	<script src="js/bundle.js"></script>
</body>
</html>
