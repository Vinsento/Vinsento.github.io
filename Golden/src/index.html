<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="initial-scale=1.0, width=device-width">
	<title>Title</title>
	<link rel="shortcut icon" type="image/x-icon" href="assets/fav/favicon.ico">
	<link rel="stylesheet" href="css/style.css">
	<script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
</head>
<body>
    
    <div class="first-screen">
        <header class="header">
            <a href="#" class="header__logo">golden</a>
            <nav class="header__nav">
                <ul class="header__list">
                    <li class="header__item">
                        <a href="#" class="header__link">home</a>    
                    </li>
                    <li class="header__item">
                        <a href="#" class="header__link">services</a>
                    </li>
                    <li class="header__item">
                        <a href="#" class="header__link">portfolio</a>
                    </li>
                    <li class="header__item">
                        <a href="#" class="header__link">about</a>
                    </li>
                    <li class="header__item">
                        <a href="#" class="header__link">contact</a>
                    </li>
                </ul>
            </nav>
        </header>
        
        <div class="first-screen__welcome">
            <p class="first-screen__text">welcome to our studio!</p>
            <h1 class="first-screen__title">itâ€™s nice to meet you</h1>
            <a href="#" class="first-screen__btn">tell me more</a>
        </div>
        
    </div>
    
    <section class="services">
        <h2 class="services__title">services</h2>
        <p class="services__subtitle">Proin iaculis purus consequat sem cure.</p>
    </section>

	<script src="js/bundle.js"></script>
</body>
</html>
